modules/prelude-go.el

```diff
@@ -61,14 +64,18 @@
     ;; gofmt on save
     (add-hook 'before-save-hook 'gofmt-before-save nil t)
 
-    ;; stop whitespace being highlighted
-    (whitespace-toggle-options '(tabs))
+    (setq company-idle-delay 0)
```

Comment:
Seems weird to have mode-specific `company-mode` configuration. Any special reason for this?

Reply:
The special reason is that I forgot to remove this 😄